build.py

```diff
@@ -109,16 +109,20 @@ def create_package_fs(build_root):
         create_dir(os.path.join(build_root, d))
         os.chmod(os.path.join(build_root, d), 0755)
 
-def package_scripts(build_root):
+def package_scripts(build_root, static=False):
```

Comment:
> Why do Windows users want static builds?

Sorry, Windows builds won't be statically compiled, but they will use the `static` flag (soon to be ` config_only`) for the packaging portion. Another reason why `static` should not have been used as the flag name. :smile: